api/SpeechGrammarList.json

```diff
@@ -0,0 +1,415 @@
+{
+  "api": {
+    "SpeechGrammarList": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/SpeechGrammarList",
+        "support": {
+          "webview_android": {
+            "version_added": null
+          },
+          "chrome": {
+            "prefix": "webkit",
+            "version_added": "33",
+            "notes": "You'll need to serve your code through a web server for recognition to work."
+          },
+          "chrome_android": {
+            "prefix": "webkit",
+            "version_added": true,
+            "notes": "You'll need to serve your code through a web server for recognition to work."
+          },
+          "edge": {
+            "version_added": null
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
+            "version_added": "44",
+            "flags": [
+              {
+                "type": "preference",
+                "name": "media.webspeech.recognition.enable"
+              }
+            ],
+            "notes": "currently speech recognition won't work on Desktop Firefox — it will be properly exposed soon, once the required internal permissions are sorted out."
+          },
+          "firefox_android": {
+            "version_added": null
```

Comment:
I did a little reading and if I understand [the story of this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1248897) properly, no part of this API is exposed to web developers on Android (as well as desktop Firefoxes), so it looks like all of the firefox_android cases can be `false` too.